scripting/togglemusic_dhook.sp

```diff
@@ -1,17 +1,11 @@
-#pragma semicolon 1
-
-#include <sourcemod>
 #include <sdktools>
 #include <sdkhooks>
 #include <dhooks>
 #include <clientprefs>
 #include <regex>
 //#include <emitsoundany>
 
-#pragma newdecls required
```

Comment:
Why would you remove the requirement to use new-style declarations?

Reply:
Which Editor do you use? I have a VS Code Plugin and SPEdit, both mark old styled Code.